tests/models/opt/test_modeling_opt.py

```diff
@@ -216,7 +216,6 @@ class OPTModelTest(ModelTesterMixin, GenerationTesterMixin, PipelineTesterMixin,
         else {}
     )
     is_encoder_decoder = False
-    fx_compatible = True
```

Comment:
I think this should not be removed

Reply:
It seems to get error for the fx tests - `test_torch_fx` and `test_torch_fx_output_loss`.
The error is happening for this [line](https://github.com/susnato/transformers/blob/74e968760c140fde0d05caad66d52ddbcb911a30/src/transformers/models/opt/modeling_opt.py#L872).

Reply:
thanks, I see, in that line can you use instead `torch.isin` ? https://pytorch.org/docs/stable/generated/torch.isin.html - suggestion from @michaelbenayoun